pkgs/top-level/aliases.nix

```diff
@@ -1595,7 +1595,7 @@ mapAliases {
   openbsdCross = openbsd; # Added 2024-09-06
 
   # LLVM packages for (integration) testing that should not be used inside Nixpkgs:
-  llvmPackages_latest = llvmPackages_19;
+  llvmPackages_latest = llvmPackages_20;
```

Comment:
We've never set this to a release candidate before, as far as I know. I think this should wait until release.

Reply:
Reverted